docs/config.md

```diff
@@ -206,14 +206,17 @@ due to lack of stable semantic convention.
 
 | ID        | Instrumented library                                                                                                            | Supported versions | Instrumentation type   | Status                                                                                                                            |
 |-----------|---------------------------------------------------------------------------------------------------------------------------------|--------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
-| `ILOGGER` | [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) **Not supported on .NET Framework** | ≥9.0.0             | bytecode or source [1] | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
-| `LOG4NET` | [log4net](https://www.nuget.org/packages/log4net)                                                                               | ≥2.0.13 && < 4.0.0 | bytecode               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `ILOGGER` | [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) **Not supported on .NET Framework** | ≥9.0.0             | bytecode or source \[1\] | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `LOG4NET` | [log4net](https://www.nuget.org/packages/log4net) \[2\]                                                                           | ≥2.0.13 && < 4.0.0 | bytecode               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 
-**[1]**: For ASP.NET Core applications, the `LoggingBuilder` instrumentation
+\[1\]: For ASP.NET Core applications, the `LoggingBuilder` instrumentation
 can be enabled without using the .NET CLR Profiler by setting
 the `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` environment variable to
 `OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper`.
 
+\[2\]: Instrumentation provides both [trace context injection](./log4net-context-injection.md)
+and [logs bridge](./internal/log4net-bridge.md).
```

Comment:
nit: shoulnd't `./internal/log4net-bridge.md` be moved to `./log4net-bridge.md`?